dev-docs/PR_PROCESS.md

```
@@ -95,7 +95,7 @@ PRs with `[WIP]` in the title are still being worked on by Copilot. Leave them a
 gh pr ready ISSUE_NUMBER
 ```
 
-### d. Update branches with master
+### d. Update branches with main
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)